### PR TITLE
finalized social login

### PR DESCRIPTION
**Pull request by supercede**
#### What does this PR do?
Integrate Facebook and Google sign in

#### Description of Task to be completed?
Currently, a new user cannot log in with their pre-existing social media accounts. He/She can only login with email password combinations.
This PR implements the feature

#### How should this manually be Tested?
- `git clone` __https://github.com/devcareer/Dae-Alright-Backend__
- `cd` **Dae-Alright-Backend**
- `git checkout <branch-name>`
- `npm install`
- `npm test`

#### What are the relevant ~~Pivotal Tracker~~ **Trello** stories?
As a user, I can log in to Barefoot application via Facebook and Google.

### CODE REVIEW

**Comment by supercede** [comment]
[As a user, I can log in to Barefoot application via Facebook and Google.](https://trello.com/c/Gro698KG/4-as-a-user-i-can-log-in-to-barefoot-application-via-facebook-and-google)

**Comment by supercede** [comment]
[As a user, I can log in to Barefoot application via Facebook and Google.](https://trello.com/c/Gro698KG/4-as-a-user-i-can-log-in-to-barefoot-application-via-facebook-and-google)

File: src/config/passport-config/facebook-passport-config.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import passport from 'passport';
+import FacebookStrategy from 'passport-facebook';
+import database from '../../database/models';
+import { findBySocialID } from './config';
+
+const { User } = database;
+
+config();
+
+passport.use(
```
[the remark this answers is not in the document]
Comment (iphyokafor) [reply src/config/passport-config/facebook-passport-config.js:11]: All these "Unexpected newline after '(' function-paren-newline", Are they eslint errors?

**Review by iphyokafor — COMMENTED** [review]
Dropped a comment..

**Pull request closed without merging**